tests/rustdoc-ui/2024-hashtag-doctest.rs

```diff
@@ -0,0 +1,14 @@
+// This test ensures that `##` are not emitting a warning when generating
+// docs with the 2024 edition.
+// Regression test for <https://github.com/rust-lang/rust/issues/136899>.
+
+//@ check-pass
```

Comment:
```suggestion
//@ check-pass
//@ edition: 2024
```

UI tests default to Rust 2015

Reply:
I'd rather test both editions at once. Neither of them should warn, after all, and the combination of features and editions should be long-term easier to manage if we make a habit of parameterizing these tests whenever there's a chance of the edition affecting it.

Reply:
Hmm, I don't fully agree (after all, `##` is Rust 2024 specific) but it doesn't hurt.